src/components/ProfilePicture.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, BoxProps, Skeleton, SxProps } from '@mui/material';
+import { useLocation } from 'react-router-dom';
+import { StyledBaseContributorIndicator } from '../pages/registration/contributors_tab/ContributorIndicator';
+import { dataTestId } from '../utils/dataTestIds';
+import { useProfilePicture } from '../utils/hooks/useProfilePicture';
+import { getContributorInitials } from '../utils/registration-helpers';
+
+interface ProfilePictureProps extends Pick<BoxProps, 'sx'> {
+  personId: string;
+  fullName: string;
+  height: string;
+  hasBorder?: boolean;
+  sx?: SxProps;
+}
+
+export const ProfilePicture = ({ personId, fullName, height, hasBorder, sx }: ProfilePictureProps) => {
+  const { profilePictureQuery, profilePictureString } = useProfilePicture(personId);
+  const isPublicPage = useLocation().pathname.includes('research-profile');
+
+  return (
+    <Box sx={{ height, aspectRatio: '1/1', ...sx }}>
+      {profilePictureQuery.isFetching ? (
+        <Skeleton variant="circular" sx={{ height: '100%' }} />
+      ) : profilePictureQuery.isSuccess ? (
+        <Box
+          component="img"
+          src={profilePictureString}
+          alt="profile-picture"
+          sx={{
+            height: '100%',
+            aspectRatio: '1/1',
+            borderRadius: '50%',
+            border: hasBorder ? '0.125rem solid black' : 'none',
+            objectFit: 'cover',
+          }}
+        />
+      ) : (
+        <StyledBaseContributorIndicator
+          sx={{
+            bgcolor: isPublicPage ? 'white' : 'primary.main',
+            color: isPublicPage ? 'primary.main' : 'primary.contrastText',
+            border: isPublicPage ? '0.125rem solid black' : 'none',
```

Comment:
Er det bedre å sende inn dette som en prop i stedet for å definere `isPublicPage` her? ser for meg at dette kan på virke ting i fremtiden når denne komponenten brukes mer. Ved at vi må endre `isPublicPage` til å fange flere eller færre paths. Da kan det være greit å ta dette i en bolsk prop allerede nå i stedet. Hadde vært greit å vite hvor mange ulike visninger det skal være da, og hvor de skal være